quinn-h3/src/server.rs

```diff
@@ -142,6 +144,32 @@ pub struct Connecting {
     settings: Settings,
 }
 
+impl Connecting {
+    pub fn into_0rtt(self) -> Result<(IncomingRequest, ZeroRttAccepted), Self> {
+        let Self {
+            connecting,
+            settings,
+        } = self;
+        let (
+            quinn::NewConnection {
```

Comment:
Style nit: maybe put the destruction of the `NewConnection` in its own statement? The low vertical density makes this complex statement pretty hard to grok.

Reply:
Simplified those and the other occurrence of this nit in a new commit.